account_asset_management/models/account_move.py

```diff
@@ -79,7 +80,15 @@ def post(self):
                     .create(vals)
                 )
                 aml.with_context(allow_asset=True).asset_id = asset.id
-        super().post()
+            refs = [
+                "<a href=# data-oe-model=account.asset data-oe-id=%s>%s</a>"
+                % tuple(name_get)
+                for name_get in move.line_ids.filtered(
+                    "asset_profile_id"
+                ).asset_id.name_get()
+            ]
+            message = _("This invoice created the asset/s: %s") % ", ".join(refs)
```

Comment:
I think we should change 'asset/s' into 'asset(s)'

Reply:
Done!